lib/vmdb/appliance.rb

```diff
@@ -10,6 +10,10 @@ def self.BUILD
       @EVM_BUILD ||= get_build
     end
 
+    def self.CODENAME
+      "Hammer".freeze
```

Comment:
why didn't you think of this for Fine?  It would have been a Fine codename.

Reply:
I am trying to Hammer it in before its too late

Reply:
@mkanoor Nailed it.